en/docs/deploy/deployment-guide.md

```diff
@@ -258,65 +231,95 @@ them.
                     clusterName = "ECS-IS-CLUSTER"
                     vpcCidrBlock = "10.0.*.*"
                     ```                    
-            Under the `clustering.properties` section, set the `region`, `clusterName`, and `vpcCidrBlock` based on 
-            the AWS ECS cluster that you created in the previous step.       
+            Under the `clustering.properties` section, set the `region`, `clusterName`, and `vpcCidrBlock` based on the AWS ECS cluster you created in the previous step.       
 
             !!! note
-                Once all the configurations are complete, build a docker image including the configurations. You can 
-                consume this docker image to 
-                create a `Task Definition` and run a new `Service` or a `Task` 
-                on the `AWS ECS cluster` that you created.
+                Once all the configurations are complete, build a docker image including the configurations. You can consume this docker image to create a `Task Definition` and run a new `Service` or a `Task` on the `AWS ECS cluster` you created.
+
+        ??? tip "Click to see the instructions for Kubernetes membership scheme"
+            When IS nodes are deployed in a clustered mode on Kubernetes, the Kubernetes Membership Scheme enables automatically discovering these servers. The Kubernetes Membership Scheme supports finding the pod IP addresses using the Kubernetes API.
+
+            - If not already present, download and copy the <a href=https://github.com/wso2/kubernetes-common/tags>kubernetes-membership-scheme-1.x.x.jar</a> to the `<IS_HOME>/repository/components/dropins/` directory.
+
+            - Configure the `<IS_HOME>/repository/conf/deployment.toml` file with the following configurations.
+
+            | Parameter | Description   | Example   |
+            |-----------|---------------|-----------|
+            | `membershipScheme`  | This is the membership scheme that will be used to manage the membership of nodes in a cluster.   | `kubernetes`    |
+            | `local_member_host` | This is the member's hostname or IP address. Set it to the pod's local IP address.   | `172.17.0.2`    |
+            | `local_member_port` | This is the TCP port used by this member and through which other members will contact this member. | `4000` |
+            | `membershipSchemeClassName` | org.wso2.carbon.membership.scheme.kubernetes.KubernetesMembershipScheme   |                |
```

Comment:
The description seems to be missing here and we seem to have the example there instead.

Reply:
We will look into these updates throuhh [1].

[1] https://github.com/wso2/product-is/issues/15117